### PR TITLE
storaged: dummy change to trigger anaconda

### PR DESCRIPTION
**Pull request by martinpitt**
Validate PR #19630

### CODE REVIEW

**Comment by martinpitt** [comment]
> ignoring unknown context fedora-rawhide-boot/cockpit-pr-19631@rhinstaller/anaconda

Ah, we need to move that too, don't we

File: pkg/storaged/index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!--
- * This file is part of Cockpit.
+ * This file is part of CockPWNED
```
Comment (allisonkarlitskaya) [inline pkg/storaged/index.html:3]: I don't like this change.  I'd prefer if we didn't proceed with it.

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

File: pkg/storaged/index.html (hunk 1 above)
Comment (martinpitt) [reply pkg/storaged/index.html:3]: Damn, someone noticed my 31337 h4x0r attempt! :see_no_evil:

**Comment by martinpitt** [comment]
Works! :+1:

**Pull request closed without merging**